pkgs/top-level/all-packages.nix

```diff
@@ -12464,6 +12466,19 @@ with pkgs;
 
   zziplib = callPackage ../development/libraries/zziplib { };
 
+  gsignond = callPackage ../development/libraries/gsignond { };
```

Comment:
Do we need to expose it? Would not something like pidgin does be cleaner?

Reply:
I thought of that, but I really wanted to just get going :). 
I think it would be cleaner too.